### PR TITLE
e4s oneapi ci: try builds that were previously failing due to binutils failure

### PR DESCRIPTION
**Pull request by eugeneswalker**
Try building packages that had previously failed due to https://github.com/spack/spack/issues/29382 which may now be resolved.

### CODE REVIEW

**Comment by eugeneswalker** [comment]
> Try building packages that had previously failed due to #29382 which may now be resolved.

Unfortunately, not resolved! Issue is specifically with `binutils +gold %oneapi` https://github.com/spack/spack/issues/29382#issuecomment-1208676420

**Comment by tldahlgren** [comment]
Looks like this is failing for one of the stacks due to broken spec (https://gitlab.spack.io/spack/spack/-/jobs/3013291):

```
==> Error: Pipeline generation failed due to the presence of the following specs that are known to be broken in develop:
[526](https://gitlab.spack.io/spack/spack/-/jobs/3013291#L526)* paraview@5.10.1%gcc@7.3.1-....
```

File: share/spack/gitlab/cloud_pipelines/stacks/e4s-oneapi/spack.yaml [hunk 1]
```diff
@@ -184,39 +188,34 @@ spack:
   # GPU
   - arborx +sycl ^kokkos +sycl +openmp std=17 +tests +examples
   - cabana +sycl ^kokkos+sycl +openmp std=17 +tests +examples
-  - kokkos-kernels build_type=Release %oneapi ^kokkos +sycl +openmp std=17 +tests +examples %oneapi
+  - kokkos-kernels build_type=Release ^kokkos +sycl +openmp std=17 +tests +examples
```
Comment (tldahlgren) [inline share/spack/gitlab/cloud_pipelines/stacks/e4s-oneapi/spack.yaml:191]: Did you intend to remove `%oneapi` here?

**Comment by rscohn2** [comment]
Some of these are enabled. The rest are blocked by issues other than binutils.

**Pull request closed without merging**